port/win32/omrsysinfo.c

```diff
@@ -951,7 +953,7 @@ omrsysinfo_get_memory_info(struct OMRPortLibrary *portLibrary, struct J9MemoryIn
 		return OMRPORT_ERROR_SYSINFO_ERROR_READING_MEMORY_INFO;
 	}
 
-	status = PdhAddCounter(statsHandle,
+	status = PdhAddEnglishCounter(statsHandle,
```

Comment:
https://github.com/eclipse-omr/omr/pull/7566#discussion_r1868245664 also needs to be applied here.